.travis.yml

```diff
@@ -1,29 +1,45 @@
+sudo: required
+
 language: go
 
 go:
   - 1.7.x
   - 1.8.x
```

Comment:
Yeah, no worries about this will do in future. Right now I'm trying to minimize number of builds ;)